android/test_app/app/src/main/java/org/pytorch/testapp/MainActivity.java

```diff
@@ -121,7 +122,7 @@ public Result(float[] scores, long moduleForwardDuration, long totalDuration) {
   @UiThread
   protected void handleResult(Result result) {
     String message = String.format("forwardDuration:%d", result.moduleForwardDuration);
-    Log.i(TAG, message);
+    Log.i(TAG, "XXX:" + message);
```

Comment:
what does "XXX" stand for? isn't TAG enough here?

Reply:
Yes, just debug info, Sorry, I have pushed land button before I checked this comment :(